tests/foreman/ui/test_errata.py

```diff
@@ -728,17 +728,14 @@ def test_positive_show_count_on_chost_page(self):
                 result = self.contenthost.fetch_errata_counts(client.hostname)
                 for errata in ('security', 'bug_fix', 'enhancement'):
                     self.assertEqual(result[errata]['value'], 0)
-                    self.assertEqual(result[errata]['color'], 'black')
```

Comment:
it make no sense to restore them grammatically from this string (the string it self compare to the numbers)
```
<i class="fa fa-warning inline-icon" ng-class="{black: host.content_facet_attributes.errata_counts.security === 0, red: host.content_facet_attributes.errata_counts.security > 0}" title="Security"></i>
```